src/Main.cpp

```diff
@@ -151,15 +154,62 @@ struct PyramidPlunder : public Game
 
 	void on_start() override
 	{
-		auto player = spawn()
-			.with<Player>(SpecialBlindfold::HumanEyes)
-			.with<Sprite>(ecs::no_entity)
-			.with<SpriteAnimation>(Spritesheet::get_by_name("pyramidplunder/archaeologist_standing"))
-			.with<Visibility>(true)
-			.with<Position>(geometry::Vec2{ 300, 100 })
-			.with<Movement>(2000.0f, 50.0f)
-			.with<KeyBinding>(KeyCode::KEY_LEFT, KeyCode::KEY_DOWN, KeyCode::KEY_UP, KeyCode::KEY_RIGHT, KeyCode::KEY_SPACE)
-			.done();
+		levelManager = new LevelManager();
+		levelManager->LoadLevel("Levels/level1.txt");
+
+		for(int i = 0; i < TILE_ROWS; i++){
+        	for(int j = 0; j < TILE_COLS; j++){
```

Comment:
All `{` should go in their own line. Same thing with if statements later in code. 

Use tab to indent second for loop little bit.

Reply:
Locally, the indentation on the second for loop is fine so I'm assuming that github is to blame for that one.